src/request-pipeline/context.ts

```diff
@@ -296,6 +299,13 @@ export default class RequestPipelineContext {
             .map(userScript => userScript.url);
     }
 
+    private async _getDestResBody (res: IncomingMessageLike | http.IncomingMessage | FileStream): Promise<Buffer> {
+        if (IncomingMessageLike.isIncomingMessageLike(res))
+            return (res as IncomingMessageLike).getBody();
```

Comment:
```suggestion
            return res.getBody();
```
(related to the suggestion below)